src/main/java/xyz/vegaone/easytrackingv2/service/UserService.java

```diff
@@ -56,4 +59,11 @@ public User updateUser(User user) {
 
         return userMapper.domainToDto(savedUserEntity);
     }
+
+    public List<User> findAllUsers(){
```

Comment:
We need an endpoint to access this method. Should have a path like `user/all`. Could you also add a unit test for this method?